### PR TITLE
attachment styling

### PR DESCRIPTION
**Pull request by Pomax**
better attachment styling, and a few fixes to deal with attachment removal

### CODE REVIEW

File: components/badges/RequirementRow.jsx [hunk 1]
```diff
@@ -86,7 +92,7 @@ var RequirementRow = React.createClass({
       return (
         <span className="attached" key={`evidence-${e.name}`}>
           {e.name}
-          <span className="fa fa-times" onClick={this.removeAttachment(e.name)}/>
+          <span className="close fa fa-times" onClick={this.removeAttachment(e.name).bind(this)} title="Remove this attachment"/>
```
Comment (gideonthomas) [inline components/badges/RequirementRow.jsx]: why `.bind`?

**Comment by gideonthomas** [comment]
@Pomax in this patch I only see the "Submit evidence" button for the first task and none of the others

**Comment by Pomax** [comment]
@gideonthomas that would be because the teach-api now requests data with x-allow-html, so the splitting regex you still have in learning (until you merge in the x-allow-html branch for learning) will not work anymore - you'll only see a single task

File: components/badges/RequirementRow.jsx (hunk 1 above)
Comment (gideonthomas) [inline components/badges/RequirementRow.jsx]: damn, ok this is fine then

**Pull request merged**